tests/test-cases/Group6-VIC-Machine/6-5-Create-Validation.robot

```diff
@@ -29,3 +29,24 @@ Suggest resources - Invalid target path
     ${output}=  Run  bin/vic-machine-linux create --name=${vch-name} --target=%{TEST_URL}/MUCH/DATACENTER --user=%{TEST_USERNAME} --image-store=%{TEST_DATASTORE} --password=%{TEST_PASSWORD}
     Should Contain  ${output}  Suggested datacenters:
     Should Contain  ${output}  vic-machine-linux failed
+
+Default image datastore
+    Log To Console  \nRunning vic-machine create - default image datastore
```

Comment:
Do we test if we can't find a datastore to use? The `else` case in the code above?